packages/components/rollup.config.mjs

```diff
@@ -1,35 +1,63 @@
 /* eslint-disable import/no-default-export */
-import multiInput from "rollup-plugin-multi-input";
+import path from "path";
```

Comment:
In general the changes in this file are as follows:

Update to be an ESM based file with the update to MJS.

Updated all 'require' calls to 'import' calls

Included modern _filename,__dirname variables as they are not available in MJS files.

Updated the glob to include all files under the `src` directory including a base level `index`

Included the nodePolyfills project so that `process` is correctly polyfilled for the `react-markdown` package.

Included a new `alias` entry for jobber/hooks. This means we no longer have to include the excludes workaround and can package jobber/hooks directly with jobber/components. This is necessary for Vite + PNPM support.

Updated the CSS Scheme to Extract instead of Inject. You will now need to include either `@jobber/components/styles` in an ESM environment or `@jobber/components/dist/styles.css` in a CJS environment in order to see Atlantis Component Styles.

Added a new ESM build to compliment the existing CJS build.

Moved the copy step until after the bundle has completed building. This ensures the index.d.ts file is present.

Removed a variety of packages from the external configuration section. In general, packages in this array should ALSO be in peerDependencies in package.json. The files being excluded from the build, was causing an issue with Vite + PNPM due to how strict they are.